pandas/core/indexes/period.py

```diff
@@ -323,7 +310,7 @@ def _int64index(self):
 
     @property
     def values(self):
-        return self.astype(object).values
+        return self._data
```

Comment:
Maybe take this opportunity to move away from `_data`?  Brainstorm: `_initvalues`, `_i8values`

Reply:
For IntervalIndex and CategoricalIndex we use `_data` as well (@jbrockmendel the reason you mention this is to not have the confusion with BlockManager `_data` of DataFrame/Series ?)

But going from our overview table

https://github.com/pandas-dev/pandas/blob/d115900f4f80d2f9016b41041bde1564980415b3/pandas/core/indexes/base.py#L690-L722

we could maybe also use the `_values` directly? Instead of having `_values` point to `_data` ?